spark/spark-tensorflow-connector/src/main/scala/org/tensorflow/spark/datasources/tfrecords/DefaultSource.scala

```diff
@@ -167,7 +167,7 @@ class DefaultSource extends DataSourceRegister
   }
 }
 
-object DefaultSource {
+object DefaultSource extends scala.Serializable {
```

Comment:
When upgrading to scala 2.12, then the `object XXX` defaultly become non serializable, which cause spark runtime error here. (because we need to broadcast the object here).